lib/iris/analysis/__init__.py

```diff
@@ -1015,6 +1015,40 @@ def post_process(self, collapsed_cube, data_result, coords, **kwargs):
         return result
 
 
+def _build_dask_mdtol_function(dask_stats_function):
```

Comment:
Personally I'd call this ``_add_mdtol_support``... Not saying you should do that, but I'm pretty confident that we can improve the decorator's name in the future... 😉